src/core/Akka/Actor/Address.cs

```diff
@@ -154,7 +155,7 @@ public bool Equals(Address other)
         }
 
         /// <inheritdoc/>
-        public override bool Equals(object obj) => obj is Address && Equals((Address)obj);
+        public override bool Equals(object obj) => obj is Address address && Equals(address);
```

Comment:
Me thinking out loud: Do we take the local here, or do we just make this:

`public override bool Equals(object obj) => Equals(obj as Address);`

Probably no real diff.

Reply:
Done